libs/agno/agno/os/managers/eval/router.py

```diff
@@ -51,4 +53,63 @@ async def get_eval_run(eval_run_id: str) -> EvalSchema:
 
         return EvalSchema.from_dict(eval_run)
 
+    @router.post("/evals/accuracy/run", response_model=EvalSchema, status_code=200)
```

Comment:
I think the format should be `POST /evals/` to create an eval? Then you specify the type in the payload?